.github/workflows/scripts/istio_deploy.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
Does one of the existing GitHub Actions not cover this use case already?

@DelusionalOptimist the patterns GH Action might be useful here. // @sayantan1413

Reply:
Our initial thought was to use patterns as well. But upon some thought, adding patterns here to deploy a mesh and onboard an application might be an overkill. Not using patterns but users bringing in their own cluster would also mean we limit the scope of this action to do one thing- running performance tests.

Reply:
The e2e workflow that uses the meshery-smp-action in the Meshery repo does not deploy a mesh and onboard an application. But it touches a lot of functionalities. It runs a performance test to an endpoint which isn't there.

Reply:
The scope of this GH Action is only that: to run performance tests. There is a separate GH Action for executing patterns.

Reply:
The e2e workflow in the Meshery repo needs to deploy a service mesh and onboard applications. Let's get this at the top of your priority list.

Reply:
"to touch and endpoint which isn't there"? What? Fix that, not create a separate workflow.

Reply:
Yes. Will include the patterns action in the workflow in the Meshery repo to handle this once it is finished.

@sayantan1413